source/configuration/persistence-config.txt

```diff
@@ -30,8 +30,9 @@ the persistence configuration of a model class.
 
 .. note::
    
-   "Client" refers to a host configuration defined under ``clients`` in your
-   ``mongoid.yml`` file. Most applications use a single client named ``default``.
+   The term client refers to a host configuration defined under
```

Comment:
i: i think you need quotes to designate it as a term, rather than something called a 'term client'. monospace might make more sense depending on context

```suggestion
   The term "client" refers to a host configuration defined under
```